proto/minder/v1/minder.proto

```diff
@@ -2001,12 +2048,67 @@ message RoleAssignment {
     string role = 1;
     // subject is the subject to which the role is assigned.
     string subject = 2;
+    // display_name is the display name of the subject.
+    string display_name = 5;
     // projectt is the projectt in which the role is assigned.
     optional string project = 4;
 
     reserved 3; // deprecated context
 }
 
+message ListInvitationsRequest {
+}
+
+message ListInvitationsResponse {
+    repeated Invitation invitations = 1;
+}
+
+message ResolveInvitationRequest {
+    // code is the code of the invitation to resolve.
+    string code = 1;
+    // accept is true if the invitation is accepted, false if it is rejected.
+    bool accept = 2;
+}
+
+message ResolveInvitationResponse {
+    // role is the role that would be assigned if the user
+    // accepts the invitation.
+    string role = 1;
+    // email is the email address of the invited user.
+    string email = 2;
+    // project is the project to which the user is invited.
+    string project = 3;
+}
```

Comment:
Should we have one more field with the invitation code for which this response is confirmed? I'm okay with not useful, so just asking.

Reply:
I'm inclined not to repeat data from the request into the response here, and the code is in the request.

To avoid seeming hypocrisy, something like "GetProject" returns a _project object_, which may happen to contain the ID of the project that's also in the request.  We "repeat" the project ID in the response because the ID is part of the _object_, and we want to transfer consistent objects (REST = representational state transfer) where possible, rather than incomplete subsets.